### PR TITLE
Upgrade jsonpointer, standard, and mocha

### PR DESCRIPTION
**Pull request by tijn**
## What Changed & Why

My goal was to resolve [this dependabot alert](https://github.com/twg/devour/security/dependabot/package-lock.json/jsonpointer/open) and while doing so I upgraded the standard package to the latest version. Then I ran `npm run autofix` to format the code according to the latest standards (this makes the PR huge; my apologies) and fixed the mocha configuration since it got updated too and its default configuration settings changed over time.

TLDR: I dragged a few libraries into the modern age and this introduced many small code changes.

### CODE REVIEW

**Review by auvipy — APPROVED** [review]

**Comment by tijn** [comment]
> the code changes seems appropriate to me. btw don't we have any CI active? if not I want to start one

@auvipy we do! You can find two badges at the top of the readme file that link to [this page on Travis](https://travis-ci.org/github/twg/devour) and [this one for Snyk[(https://snyk.io/test/github/twg/devour). Travis got sold a few years ago to some company that is not doing much with it though, so I'm all for replacing it with another CI solution if you know a better one.

**Comment by tomers** [comment]
LGTM
